code/modules/mob/living/living.dm

```diff
@@ -307,14 +307,14 @@
 		log_combat(src, M, "grabbed", addition="passive grab")
 		if(!supress_message && !(iscarbon(AM) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER))) //Everything in this if statement handles chat messages for grabbing
 			var/mob/living/L = M
-			if (L.getorgan(/obj/item/organ/tail) && zone_selected == BODY_ZONE_PRECISE_GROIN) //Does the target have a tail?
+			if (L.getorgan(/obj/item/organ/tail) && (is_zone_selected(BODY_ZONE_PRECISE_GROIN, precise_only = TRUE) || is_zone_selected(BODY_GROUP_LEGS))) //Does the target have a tail?
```

Comment:
does this work?? won't this make precise (old system) leg selections also target the tail?

Reply:
Yes, but I can see why this would be confusing